doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -18,17 +18,24 @@ For more information regarding AWS FOTA, see the documentation for the :ref:`lib
 .. note::
    The AWS IoT library and the :ref:`lib_aws_fota` library share common steps related to `AWS IoT console`_ management.
 
+Requirements
+************
+
+To use this library, you must build and program the :ref:`AWS FOTA <aws_fota_sample>` sample, available for the nRF9160 DK.
```

Comment:
```suggestion
To use this library, you must build and program the :ref:`AWS FOTA <aws_fota_sample>` sample available for the nRF9160 DK.
```

Reply:
```suggestion
To use this library, you can build and program the :ref:`AWS IoT <aws_iot>` sample, available for the nRF9160 DK.
```

Reply:
This does not sound right. My guess is something to this direction:
"You must enable this library to be able to use (or build and program?) the :ref:`aws_iot` sample available for..."

And it still feels a bit weird, because we usually state something about enabling a library in the sample docs. Not like here. The AWS IoT sample already mentions that you would need to enable this library to be able to build and program the sample.

Reply:
Yeah..may be we do not need this at all here?

Reply:
Maybe "This library is meant for use with the <sample>"?

Reply:
@simensrostad could you suggest?

Reply:
Yes. The idea is that the AWS IoT sample showcases the use of the AWS IoT library.